Set-1/Solved Programs/BrainCar/TestCaseForBrainCar.java

```diff
@@ -0,0 +1,29 @@
+package com.braincar;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+
+public class TestCaseForBrainCar {
+
+	@Test

```

Comment:
Comments for test missing. What does this test actually test?

Reply:
Test annotation tells JUnit that the public void method to which it is attached can be run as a test case. To run the method.